### PR TITLE
Format path parameter names

### PR DESCRIPTION
**Pull request by aneeshafedo**
## Purpose

> $subject  
- This PR also includes fixes for below minor bugs. 
> Remove extra white space in import statements
> Let users to decide on overriding test files

Resolves https://github.com/ballerina-platform/ballerina-openapi/issues/442
Resolves https://github.com/ballerina-platform/ballerina-openapi/issues/423

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    public isolated function init(http:ClientConfiguration clientConfig =  {}, string serviceUrl = "localhost:9090/payloadV") returns error? {
+        http:Client httpEp = check new (serviceUrl, clientConfig);
+        self.clientEp = httpEp;
+    }
+    #
+    # + 'version - Version Id
+    # + versionName - Version Name
+    # + return - Ok
+    remote isolated function operationId04(int 'version, string versionName) returns string|error {
+        string  path = string `/v1/${'version}/v2/${'versionName}`;
```
Comment (LakshanSS) [inline openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal:20]: Check formatting.
There should be only one space between `string` and `path`

File: openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import ballerina/http;
```
Comment (LakshanSS) [inline openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal:1]: Licence header :)

File: openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+# + clientEp - Connector http endpoint
+public client class Client {
+    http:Client clientEp;
+    # Client initialization.
+    #
+    # + clientConfig - Client configuration details
+    # + serviceUrl - Connector server URL
+    # + return - Returns error at failure of client initialization
+    public isolated function init(http:ClientConfiguration clientConfig =  {}, string serviceUrl = "localhost:9090/payloadV") returns error? {
+        http:Client httpEp = check new (serviceUrl, clientConfig);
```
Comment (LakshanSS) [inline openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal:12]: Shall we use a better name than `httpEp`

File: openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal (hunk 2 above)
Comment (aneeshafedo) [reply openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal:1]: These are sample files used in test cases. Adding license header is a separate command line options and there are set of samples to check the license header generation.

File: openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal (hunk 3 above)
Comment (aneeshafedo) [reply openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal:12]: This code is auto generated.

File: openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal (hunk 1 above)
Comment (aneeshafedo) [reply openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal:20]: Please note that these are only the sample files used only for test scenarios where the formatting is anyway removed to to compare the generated ballerina code and expected ballerina code.

**Review by indikasampath2000 — APPROVED** [review]

**Pull request merged**